tests/installation/test_chef.py

```diff
@@ -73,3 +73,34 @@ def test_get_cache_directory_for_link(config):
     )
 
     assert expected == directory
+
+
+def test_no_backslashes_with_file_protocol(config, mocker):
```

Comment:
You should instead probably just updated the existing test case to verify the issue. Like so;

```py
def test_get_cached_archives_for_link(config, mocker):
    chef = Chef(
        config,
        MockEnv(
            marker_env={"interpreter_name": "cpython", "interpreter_version": "3.8.3"}
        ),
    )

    distributions = Path(__file__).parent.parent.joinpath("fixtures/distributions")
    mocker.patch.object(
        chef,
        "get_cache_directory_for_link",
        return_value=distributions,
    )

    archives = chef.get_cached_archives_for_link(
        Link("https://files.python-poetry.org/demo-0.1.0.tar.gz")
    )

    assert archives
    assert set(archives) == {
        Link(path.as_uri())
        for path in distributions.glob("demo-0.1.0*")
    }
```

Reply:
Done.